src/app/models/api-models/user-gender/user-gender.ts

```diff
@@ -0,0 +1,53 @@
+import {Language} from '../language/language';
+import {LanguageFactory} from '../language/language';
+
+// API attribute interfaces
+interface UserGenderApiAttributes {
+  id: string;
+  name: string;
+  language: Language; // TODO missing relation in docs?
+  languageId: string; // TODO missing in docs?
```

Comment:
Yeah thats what i thought aswell. All with translated_text could have a language_id and a language relation to make it consistent @buren 😄